distributed/scheduler.py

```diff
@@ -4082,6 +4084,105 @@ def _to_dict(
         info.update(recursive_to_dict(extra, exclude=exclude))
         return info
 
+    async def get_cluster_state(
+        self,
+        exclude: Collection[str],
```

Comment:
This syntax is failing on 3.8 of course:
```python-traceback
  File "distributed/scheduler.py", line 4089, in init distributed.scheduler
    exclude: Collection[str],
TypeError: 'ABCMeta' object is not subscriptable
```
@jakirkham can we add `from __future__ import annotations` to `scheduler.py`, or will that mess with Cython?

xref https://github.com/dask/distributed/issues/5685 for an example of the contributor/maintenance burden of using type annotations for Cythonization